kindlecomicconverter/comic2ebook.py

```diff
@@ -1530,6 +1608,14 @@ def makeBook(source, qtgui=None):
             else:
                 filepath.append(getOutputFilename(source, options.output, '.cbz', ''))
             makeZIP(tome + '_comic', os.path.join(tome, "OEBPS", "Images"))
+        elif options.format == 'PDF':
+            print("Creating PDF file with PyMuPDF...")
+            # determine output filename based on source and tome count
+            suffix = (' ' + str(tomeNumber)) if len(tomes) > 1 else ''
```

Comment:
Why are you using tomeNumber? We aren't splitting PDF's

Reply:
But we are 'splitting' CBR/CBZ and then merging them together

Reply:
If we had a PDF (200 mb limit it would come up)